packages/story-editor/src/components/footer/carousel/skeletonPage.js

```diff
@@ -58,8 +59,10 @@ function SkeletonPage({ pageId, index }) {
 
   const bgElement = page.elements[0];
   const { isMedia } = getDefinitionForType(bgElement.type);
+  // Using isHexColorString for extra hardening.
+  // See https://github.com/google/web-stories-wp/issues/9888.
   const bgColor =
-    isMedia && bgElement.resource?.baseColor
+    isMedia && isHexColorString(bgElement.resource?.baseColor)
```

Comment:
Hm, I suppose migration wasn't fully working that for some elements it's still not hex?

Reply:
Still trying to figure that one out

See https://github.com/google/web-stories-wp/issues/9888#issuecomment-986664131 and #9897